gems/aws-sigv4/sig/credentials.rbs

```diff
@@ -0,0 +1,30 @@
+module Aws
+  module Sigv4
+    class Credentials
```

Comment:
Looks like a private API.
https://github.com/aws/aws-sdk-ruby/blob/8c6f65feb0500775b50153e0c24a56381509c873/gems/aws-sigv4/lib/aws-sigv4/credentials.rb#L5-L8

Reply:
Sorry, I deleted it.
a72b3bcfb6b8ab60ae2fd5cb968b861c7f02fe84